components/engine_rocks/src/raft_engine.rs

```diff
@@ -53,7 +51,7 @@ impl RaftEngine for RocksEngine {
                 }
                 let key = keys::raft_log_key(region_id, i);
                 match self.get_value(&key) {
-                    Ok(None) => return Err(Error::Storage(StorageError::Unavailable)),
+                    Ok(None) => return Err(Error::Engine("The Log has been compacted".into())),
```

Comment:
Should be `StorageError::Unavailable`.

Reply:
How about add `Error::RaftEngine`?

Reply:
Should be `DataUnavailable` or `EntriesUnavailable`.

Reply:
Ok